src/frontends/tensorflow/src/pass/transpose_sinking.cpp

```diff
@@ -105,6 +119,10 @@ static void insert_transpose(const shared_ptr<Node>& target, const shared_ptr<No
                    << " input index " << input_index;
 
     target->input(input_index).replace_source_output(new_transpose->output(0));
+    if (std::dynamic_pointer_cast<Result>(target)) {
+        new_transpose->output(0).add_names(arg.get_names());
```

Comment:
Why this is important only if `target` is Result? Shouldn't it be true for any case, except maybe when `arg` is `Parameter`??

Reply:
1. because in this case we don't need to swap these names in this case
![image](https://user-images.githubusercontent.com/21181928/201397344-646d35b3-3e89-4cbb-880a-7a7d51098522.png)
2. I don't see any reasons to swap tensor names of Node1 output and Transpose output in this case.
They never affect network outputs. Do you have any compelling reasons for that?
![image](https://user-images.githubusercontent.com/21181928/201398921-2b1fd7ff-aaad-4ac8-b576-b37be48a7a3b.png)